src/databricks/labs/ucx/cli.py

```diff
@@ -705,8 +705,14 @@ def migrate_acls(
         workspace_contexts = [ctx]
     else:
         workspace_contexts = _get_workspace_contexts(w, a, run_as_collection, **named_parameters)
-    target_catalog, hms_fed = named_parameters.get("target_catalog"), named_parameters.get("hms_fed", False)
+    target_catalog, hms_fed, dry_run = (
+        named_parameters.get("target_catalog"),
+        named_parameters.get("hms_fed", False),
+        named_parameters.get("dry_run", False),
+    )
```

Comment:
```suggestion
    target_catalog = named_parameters.get("target_catalog")
    hms_fed = named_parameters.get("hms_fed", False)
    dry_run = named_parameters.get("dry_run", False)
```
nit. every line is a liability, make them less and simpler

Reply:
Done